tests/glsl-intrinsic/shader-subgroup/shader-subgroup-builtin-variables.slang

```diff
@@ -0,0 +1,34 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+//TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl -DTARGET_HLSL
+//TEST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda -DTARGET_CUDA 
+//TEST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp
+
+// currently unused but required to be implemented once gl_NumSubgroups etc.. have functionality implemented
+//T-EST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl
+//T-EST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl -emit-spirv-directly
```

Comment:
Shouldn't this work even when slang may not have implementations for the variables, when slangc generates GLSL source file, glslc should be able to properly compile them, isn't it?

Reply:
I can compile them, in an earlier message today I was asked to remove them since it would confuse people into believing we are testing functionality which we are not really doing any of